CONTRIBUTING.md

```diff
@@ -1,55 +1,156 @@
-# Contributing to Real-time Ubuntu docs
+# Contributing to Real-time Ubuntu documentation
```

Comment:
I adopted the latest template from the starter pack and incorporated the original Contributing guide in the RTU repo.